### PR TITLE
feat: New SSO workflow skeleton

### PR DESCRIPTION
**Pull request by marlonkeating**
[Jira Ticket](https://2u-internal.atlassian.net/browse/ENT-7585)

Placeholder screens for the new SSO Integration workflow.

image

image

image

image

# For all changes

- [X] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [X] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

# Remaining Work

- [x] Change edX Service Provider Metadata button color to Primary
- [x] Show SAP fields on page 2, which will be selectively hidden later
- [x] Add selectively hidden metadata fields on page 1
  - [x] Hidden metadata fields unit tests
- [x] Add back buttons on pages 2+

### CODE REVIEW

**Review by alex-sheehan-edx — COMMENTED** [review]
A couple thoughts from manual testing, happy to discuss:
- After the first step, I think we need a back button for all the other steps instead of the cancel button
- Can we remove the save on cancel language from the cancel modal
- Can we center the stepper elements to match figma
- Can you add all the fields to step two (not just the SAP ones)? My thinking is that once we have the config context we can hide/show the sap fields depending on the external idp
- Can you add the behavior of selecting the idp metadata radios in step one reveal the field (in the case of metadata url) and the file upload (in the case of xml file)? No need for the full file upload, hopefully it can just be a mock implementation?

File: src/components/settings/SettingsLMSTab/LMSConfigPage.jsx [hunk 1]
```diff
@@ -37,6 +37,7 @@ const LMSConfigPage = ({
   return (
     <div>
       <FormContextWrapper
+        workflowTitle="New learning platform integration"
```
[the remark this answers is not in the document]
Comment (marlonkeating) [reply src/components/settings/SettingsLMSTab/LMSConfigPage.jsx:40]: The FormWorkflow originally had this hard-coded, as it was only used in the LMS config stepper.

File: src/components/settings/SettingsSSOTab/steps/NewSSOConfigConfigureStep.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+          <ValidatedFormControl
+            formId="DummyDisplayName"
+            type="text"
+            floatingLabel="Display Name (Optional)"
+            fieldInstructions="Create a custom display name for this SSO integration"
+          />
+        </Form.Group>
+
+        {/* TODO: Render SAP fields selectively once logic is in place */}
+        {renderNonSAPFields()}
```
Comment (alex-sheehan-edx) [inline src/components/settings/SettingsSSOTab/steps/NewSSOConfigConfigureStep.tsx:123]: you could probably call this "base fields" or something to that extent

**Review by alex-sheehan-edx — APPROVED** [review]
Looks awesome

**Pull request merged**